### PR TITLE
chore: ignore changelog in prettier

### PR DESCRIPTION
**Pull request by KATT**

### CODE REVIEW

**Comment by KATT** [comment]
cc @JoshuaKGoldberg

File: CHANGELOG.md
Comment (KATT) [inline CHANGELOG.md:1]: updated it so it's consistently "wrong" / what the release script does

Comment (JoshuaKGoldberg) [reply CHANGELOG.md:1]: Huh, if I run `pnpm format --write` locally then nothing changes in `CHANGELOG.md`. What's going on?

Comment (KATT) [reply CHANGELOG.md:1]: You can see here https://github.com/KATT/tupleson/commit/f739b1b68e4eb1b2ae3ba29d536b3635e59c1f34

Comment (JoshuaKGoldberg) [reply CHANGELOG.md:1]: What weirds me out is that I can't get the file to be formatted in that weird way (extra newlines, etc.)... e.g. https://github.com/JoshuaKGoldberg/create-typescript-app/blob/a22ffd97f7b0edf1a76b2d9d37c4f61c99eda972/CHANGELOG.md doesn't have them. 🤔

**Pull request merged**